packages/config/config/devices/0x0063/ge_46202.json

```diff
@@ -1,42 +1,63 @@
 // Jasco Products GE 46202
 // GE Enbrighten Z-Wave Plus Smart Switch
 {
-	"manufacturer": "Jasco Products",
-	"manufacturerId": "0x0063",
-	"label": "GE 46202",
-	"description": "GE Enbrighten Z-Wave Plus Smart Switch",
-	"devices": [
-		{
-			"productType": "0x4952",
-			"productId": "0x3137"
-		}
-	],
-	"firmwareVersion": {
-		"min": "0.0",
-		"max": "255.255"
+    "_approved": true,
```

Comment:
Should _approved be here? I don't see it in the other files

Reply:
Not sure, it was in some of them. I didn't add it.

Reply:
I think it might have been a merge artifact then as it is showing you adding it

Reply:
It is. I didn't change anything outside the association and compat sections, except for the formatting I just cleaned up.